### DISCUSSION TITLE
How does session sharing work?

### DISCUSSION THREAD

**Discussion by ghost** [zellij-org/zellij#1276] [Q&A] [answered]
Talking about the feature "Share sessions with others" that's mentioned in the README's Roadmap.

This is basically live screensharing for the terminal, right? Or am I mistaken?

How does one access this feature? I can't find anything in the Docs.

**Comment by imsnif** [comment, marked as answer]
There's more information here: https://zellij.dev/news/multiplayer-sessions/

Would definitely be a good idea to add it to the documentation.

**Comment by mickg10** [comment]
Does this work if people are on the same machine, but under different usernames?

Thanks!

**Comment by imsnif** [comment]
Unfortunately not @mickg10 !
This is intentionally because when implementing this I was afraid of the security concerns (eg. run zellij as a user with sudo privileges, connect to their session from a user without). There are certainly ways around this, but more time and thought needs to be given to them.
